src/login.rs

```diff
@@ -248,7 +262,23 @@ impl Login {
                     Some(&*imp.phone_number_entry),
                 );
             }
-            AuthorizationState::WaitCode(_) => {
+            AuthorizationState::WaitCode(data) => {
+                imp.code_page
+                    .set_description(Some(&if data.code_info.timeout > 0 {
+                        gettext!(
+                            "The code should arrive to you via {} within the next {} seconds.",
```

Comment:
Mh, it seems that the [timeout](https://docs.rs/tdlib/latest/tdlib/types/struct.AuthenticationCodeInfo.html#structfield.timeout) property here indicates when a new code can be re-sent, not when the current code will be received (because the code should actually be received almost instantaneously when this update arrives). So I guess this string is not needed.